content/post/GSOC_AMA_report.md

```diff
@@ -0,0 +1,111 @@
+
+<iframe width="560" height="315" align=center src="https://www.youtube.com/embed/QVnN34YGz8s" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+**Selected students** get a stipend, loads of experience and skills working under professional mentors. **ALiAS AMA program** has been all about mentoring our members and whosoever that is interested in learning about open-source and willing to contribute to it.
+
+We invited **GSoC mentors**, **admins** and **students** from every major organisation in our networks to get their insights, experience and most importantly answer questions. Our ideology is that **no question is dumb**. And when you believe in this, that's when you know that one could be truly open in this environment.
+
+Henceforth, we had **15 mentors**, for **15 days** straight. For a streak of webinars, live sessions, daily meetups, proposal reviews, chats, doubt clearing, and motivational AMA's.
+
+**AMA (Ask Me Anything) sessions** from the very beginning, have been completely **open** with people coming from together to help, mentor and participate in the program resulting in a nine times the selection this year compared to last 3 years! **This was only possible because of our beliefs, teamwork and true hard work.**
+
+> Be the change you want to see in the world or at least try...
+
+## Activities Related to GSoC done by ALiAS:
+
+###  **Mentoring Sessions**
+
+- Mentoring students throughout the year through various channels like WhatsApp Members Groups, Workshops, Open Meetups, Casual Meetups, and more.
+- Special Sessions by:
+        1.  **Shivam Rajput** (2017 GSoC Student and Amity Alumni)
+        2.  **Shashank** (Debian GSoC Student)
+        3.  **Raghav Puri** (Organization Admin and Mentor, AOSSIE Google Code-In, and GSoC 2019)
+        4.  **Aunvrat Parashar** (Founder, ALiAS)
+        5.  **Shyam Saini** (GSoC Student 2017)
+        6.  **Vipul Gupta** (GSoC Student 2018, ALiAS founding member),
+
+### **AMA Sessions**
+
+ALiAS Conducted **15 days 15 sessions of Ask Me Anything, on GSoC** in the last two weeks before GSoC Proposal Deadline, to help out students and guide them on various aspects of GSoC.
+
+### **Webinars**
+
+*   **Drafting the Perfect Proposal with [realslimshanky](https://twitter.com/realslimshanky) | ALiAS**[GSoC'19 Live\]
+*   **GSoC [Ask Me Anything with Raghav Puri](https://www.youtube.com/watch?v=0W9BuKwm\_h4)**
+
+<iframe width="560" align=center height="315" src="https://www.youtube.com/embed/HlvI9lX2U0g" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+The joint efforts of various **students**, **mentors**, **leaders**, **trainers**, **volunteers**, and **contributors** have led us to this proud moment. *ALiAS has always been helping students contribute to open source and will continue to do so.*
+
+## Social Media Reach
+<p align=center>
```

Comment:
Not centered, not for me
but the other 2 tweets are
![Screenshot from 2019-06-02 22-18-28](https://user-images.githubusercontent.com/30147327/58764423-6220ba00-8584-11e9-9e78-0e6d98d7e989.png)

Reply:
Remove centered, I want them left aligned only.